include/searchresultslistformaction.h

```diff
@@ -29,6 +30,10 @@ class SearchResultsListFormAction : public ItemListFormAction {
 	const std::vector<KeyMapHintEntry>& get_keymap_hint() const override;
 
 	void add_to_history(const std::shared_ptr<RssFeed>& feed, const std::string& str);
+	void set_head(const std::string& s,
+		unsigned int unread,
+		unsigned int total,
+		const std::string& url);
```

Comment:
This probably has to be marked `virtual` in the base class, and `override`n here.